docs/file-reference.md

```diff
@@ -0,0 +1,368 @@
+# OpenComposition file reference
+
+Each file defines one micro-service, which forms one `pod` controlled by it's
+controller(right now the default controller is `deployment`).
+
+
+A example using all the keys added in OpenComposition(not all keys from Kubernetes API are
+included):
+
+```yaml
```

Comment:
this example looks really scary :-( 
It might be better to have something short and easy at the beginning